src/core/stream/period/get_adaptation_switch_strategy.ts

```diff
@@ -34,7 +34,8 @@ const { ADAPTATION_SWITCH_BUFFER_PADDINGS } = config;
 export type IAdaptationSwitchStrategy =
   { type: "continue"; value: undefined } |
   { type: "clean-buffer"; value: Array<{ start: number; end: number }> } |
-  { type: "needs-reload"; value: undefined };
+  { type: "needs-reload"; value: undefined } |
+  { type: "needs-buffer-flush"; value: undefined };
```

Comment:
You added that one but never returned it...

Reply:
yes, I removed that adaptation strategy, thanks!